# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../opensp/gcc34.patch
# Copyright (C) 2004 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual gcc-3.4 fixes.

  - Rene Rebe <rene@exactcode.de>

--- OpenSP-1.5.1/include/RangeMap.h.orig	2004-05-05 19:37:38.126740944 +0200
+++ OpenSP-1.5.1/include/RangeMap.h	2004-05-05 19:38:13.464368808 +0200
@@ -8,6 +8,8 @@
 #include "Boolean.h"
 #include "ISet.h"
 #include "types.h"
+#include "constant.h"
+
 #include <stddef.h>
 
 #ifdef SP_NAMESPACE
